src/components/create-post.jsx

```diff
@@ -57,7 +59,7 @@ export default class CreatePost extends React.Component {
     const isShiftPressed = e.shiftKey;
     if (isEnter && !isShiftPressed) {
       e.preventDefault();
-      if (!this.state.isFormEmpty && !this.props.createPostViewState.isPending) {
+      if (!this.state.isFormEmpty && !this.state.attachmentQueueLength && !this.props.createPostViewState.isPending) {
```

Comment:
this.state.attachmentQueueLength is not a boolean, correct check would be `this.state.attachmentQueueLength == 0`